src/main/resources/application.properties

```diff
@@ -6,6 +6,7 @@ spring.data.mongodb.database=gamification
 spring.data.mongodb.repositories.enabled=true
 spring.data.mongodb.uri=mongodb://localhost:27017/gamification
 keepers.baseURL=http://localhost:8090
-endpoint.keepers=/keepers
+endpoint.keepers=/v1/keepers
 spring.output.ansi.enabled=ALWAYS
-cron.expression=* 0 0 * * SUN
\ No newline at end of file
+cron.expression=* 0 0 * * SUN
+#Schedueled task will run every Sunday at 00-00
```

Comment:
Comments always should be written above code they describe. Put it up to one string.